### PR TITLE
PR for branch add-remove-edit-task : To-do-List

### PR DESCRIPTION
**Pull request by anita00001**
In this branch, we have implemented the following features:

- [ ] Removed all hardcoded items from the tasks array
- [ ] Created a new JavaScript file for the new functionality
- [ ] Implemented a function for adding a new task (add a new element to the array).
- [ ] Implemented a function for deleting a task (remove an element from the array).
- [ ] Implemented a function for editing task descriptions.

Other additional Features:
- [ ] By default new tasks have the property `completed: false` and the property `index: new-array.length`
- [ ] Deleting a task updates all remaining items' indexes, so they represent the current list order and are unique
- [ ] All changes to the To-do-List are saved in local storage

### CODE REVIEW

File: src/localstorage.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  // add function
+  addToDo(description) {
+    const completed = false;
+    const index = this.taskList.length;
+    const updatedToDo = [
+      ...this.taskList,
+      { completed, description, index },
+    ];
+    this.updateStorage(updatedToDo);
+  }
```
Comment (TedLivist) [inline src/localstorage.js]: - [x] Kindly note that `index` values of the tasks should correspond with their locations in the array starting from 1. So the first tasks should have an index of 1, and the second one should be 2 and so on. But currently your `index` start at 0. Kindly fix that.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @anita00001,

Good job so far!
✔️ Index values of the remaining tasks are updated correctly after deletion
✔️ Implementation of tasks edit functionality
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/localstorage.js (hunk 1 above)
Comment (anita00001) [reply src/localstorage.js]: @TedLivist Thank you so much for pointing this out.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @anita00001,

✔️ Tasks `index` values reflect positions in the array

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**